roles/software_engineer_2.md

```diff
@@ -2,17 +2,61 @@
 
 Bristol, Manchester & London, UK.
 
-Our Engineers build software that makes our customers happy, they prefer problem solving over task completing, love learning from colleagues, and constantly strive to be nice humans.
+Please apply for this role at [www.madetech.com/careers](https://www.madetech.com/careers).
+
+Our Engineers deliver digital, data and technology outcomes that improve society. They do this by delivering and architecting software.
 
 ## What does the job entail?
 
-We primarily write and deliver custom software for the public sector. We work across central and local government, as well as in health, and our past lies in the technology startup world. Technical excellence for us isn’t about delivering to feature lists. We place a strong emphasis on outcome-based delivery; ensuring our customer’s goals are understood and achieved with the technology we deploy.
+At Made Tech we want to positively impact the future of the country by using technology to improve society. We help public sector organisations deliver quality software to help citizens get more from public services. To do this we work alongside brilliant public servants to modernise technology and accelerate digital delivery.
+
+As an Engineer 2 you will collaboratively lead planning of work, seeking feedback and support when needed, influence the approach taken to deliver features for our customers and their users, and will be comfortable managing complex changes within a single system.
```

Comment:
I'm not sure this directly lines up with Engineer 2 expectations, was it copied from a more senior role?

Reply:
Yea happy for stuff to be pulled out and adapted - we are looking to hire Engineer 2s now, and I figured I'd expect them to have the same type of experience as the Academy - so I've softened some of the expectations, happy for others to be addressed and advice on this too.

Reply:
Oh, additionally my understanding is we are looking to hire on the higher end of Engineer 2, so would expect more experience, i.e. on track to Senior I think. But be great to get clarified

Reply:
Sure thing - it's fine to tweak to reflect our current open positions, I think. Better to accurately reflect the E2 roles we're hiring for, see comment about desirable experience (maybe we keep the list longer in that case)